db/db-challenge3/db-008/db-008.sql

```diff
@@ -0,0 +1,11 @@
+SELECT u.name
```

Comment:
<修正不要>
今回の要件が、下記でした。さとうさんの場合、nameというカラム名がかぶってわかりにくいので、最終的な表示部分だけ、ASを使っても良いと思います。

```
　・参加しているユーザー名、参加しているチャットルーム名、参加日時を表示すること
```

```suggestion
SELECT u.name AS "ユーザー名"
```

Reply:
ご指摘ありがとうございます！
以後気をつけます！